cmd/create/execution_test.go

```diff
@@ -147,20 +180,65 @@ func TestCreateLaunchPlanExecutionFunc(t *testing.T) {
 	tearDownAndVerify(t, `execution identifier project:"flytesnacks" domain:"development" name:"f652ea3596e7f4d80a0e"`)
 }
 
-func TestCreateTaskExecutionFunc(t *testing.T) {
+func TestCreateRelaunchExecutionFunc(t *testing.T) {
 	setup()
 	createExecutionSetup()
-	executionCreateResponseTask := &admin.ExecutionCreateResponse{
+	executionCreateResponseLP := &admin.ExecutionCreateResponse{
 		Id: &core.WorkflowExecutionIdentifier{
 			Project: "flytesnacks",
 			Domain:  "development",
-			Name:    "ff513c0e44b5b4a35aa5",
+			Name:    "f652ea3596e7f4d80a0e",
 		},
 	}
-	mockClient.OnCreateExecutionMatch(ctx, mock.Anything).Return(executionCreateResponseTask, nil)
-	executionConfig.ExecFile = testDataFolder + "task_execution_spec.yaml"
+	literalMap := &core.LiteralMap{
+		Literals: nil,
```

Comment:
nit: do you need to bother initializing to nil?

Reply:
Yeah not required. I had some params earlier but not required for the test and hence removed it